tests/test_project.py

```diff
@@ -669,6 +682,738 @@ def helper_test_list_thermal_maps(project):
             pytest.fail(str(e.str_itr()))
 
 
+def helper_test_update_thermal_maps(project):
```

Comment:
All of these tests contain the `file_comment` field in the `thermal_map`. Since it is documented as optional, there should be at least one test that doesn't include it.

Reply:
Sounds good. I erased the fileComment of the last test.